homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/index.html

```diff
@@ -4,20 +4,45 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>About</title>
-        <link href="./style.css" rel="stylesheet" type="text/css">
+        <link href="./styles/style.css" rel="stylesheet" type="text/css">
     </head>
     <body>
-        <section>
-            <article>
-                <h1>About me</h1>
-                <p>
-                    Hi! My name is Oleksandr and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.<br>
-                    This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023>.</a>
-                    <br>
-                    This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio.<br>
-                    You can contact me via @m0oooony and/or check out my GitHub.
-                </p>
-            </article>
-        </section>
+        <header class="header">
+            <div class="header_flexbox">
+                <img src="./images/logo.svg" class="logo" alt="logo">
+                <nav>
+                    <ul class="nav_items">
+                        <li class="nav_item">
+                            <a href="./games.html">Games</a>
+                        </li>
+                        <li class="nav_item">
+                            <a href="./about.html">About</a>
+                        </li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <main>
+            <section class="about_container">
+                <div class="section_image">
+                    <img src="./images/about-workplace.svg" class="workplace_image" alt="workplace">
+                </div>
+                <article class="section_text">
+                    <h1>About me</h1>
+                    <p>
+                        Hi! My name is Oleksandr and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
+                    </p>
+                    <p>
+                        This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023></a>.
```

Comment:
Please remove `>` from linkt text: Masters Academy in 2023**>**
I'm approving your pr, so do it in the next task as well